lib/screens/gallery_screen.dart

```diff
@@ -35,20 +37,22 @@ class GalleryScreen extends StatelessWidget {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: EmptyAppBar(AppLocalizations.of(context).gallery),
-      floatingActionButton: FloatingActionButton(
-        child: const Icon(Icons.add),
-        onPressed: () {
-          Navigator.pushNamed(
-            context,
-            FormScreen.routeName,
-            arguments: FormScreenArguments(
-              AppLocalizations.of(context).addImage,
-              ImageForm(),
-              hasListView: true,
+      floatingActionButton: (Platform.isLinux || Platform.isMacOS)
```

Comment:
one last question, why do you hide the floating action buttons on the desktop? Currently (at least on a couple of screens) this is the only way to add new entries. For other checks (like showing videos or using the scanner e.g.) I understand that they are still problematic on those platforms

Reply:
I believe this one is because `image_picker` is not available for Linux or for macOS, and the button leads to it.

It wouldn't make sense to have a button lead nowhere, and rewriting this functionality to work with a different library that does support Linux and macOS would require some time, which I don't have right now.

Reply:
ahh, the image picker, hadn't thought of that, you're right